### PR TITLE
docs(plugins): update readme for deprecation

### PR DESCRIPTION
**Pull request by scefali**

### CODE REVIEW

**Comment by joshuarli** [comment]
LGTM, but I feel like @BYK should give the approval. Also, this might be a good time to standardize our repository deprecation and archival process, if there isn't one written down already.

File: README.rst [hunk 1]
```diff
@@ -1,38 +1,6 @@
...
-
-These plugins have been replaced with Sentry's built in `Global Integrations <https://docs.sentry.io/workflow/integrations/global-integrations/>`_.
-
-* `Slack <src/sentry_plugins/slack/README.rst>`_
-* `GitHub <src/sentry_plugins/github/README.rst>`_
-* `Gitlab <src/sentry_plugins/gitlab/README.rst>`_
-* `Bitbucket <src/sentry_plugins/bitbucket/README.rst>`_
-* `Visual Studio Team Services <src/sentry_plugins/vsts/README.rst>`_
-* Jira
+This repo is no longer in use. All plugins have been permanently moved into https://github.com/getsentry/sentry
```
Comment (BYK) [inline README.rst]: Would be good to link to where these plugins live in that repo.

**Review by BYK — APPROVED** [review]
We're also gonna put this repo into archive mode which says the following:
```
 This will make the getsentry/sentry-plugins repository, issues, pull requests, labels, milestones, projects, wiki, releases, commits, tags, branches, reactions and comments read-only and disable any future comments. The repository can still be forked.

Ensure you’ve changed any repository settings, considered closing all open issues and pull requests and updated your README before you archive this repository.

Once archived, you can unarchive the repository at any time.
```

Shall we go ahead and close all issues and pull requests with proper messaging before this?

File: README.rst (hunk 1 above)
Comment (scefali) [reply README.rst]: @BYK Makes sense. That location is about to change so I will wait until that happens before updating and merging this PR.

**Comment by JTCunning** [comment]
Merging because of Percy expectation.

**Pull request merged**